openhtf/util/validators.py

```diff
@@ -75,9 +76,30 @@ def create_validator(name, *args, **kwargs):
 
 _identity = lambda x: x
 
+
+class ValidatorBase(object):
```

Comment:
You didn't mention that it doesn't have to be a class, and the only example is of a class